src/main/java/io/kestra/storage/s3/S3Storage.java

```diff
@@ -99,14 +146,50 @@ public Long lastModifiedTime(String tenantId, URI uri) throws IOException {
         }
     }
 
+    @Override
+    public FileAttributes getAttributes(String tenantId, URI uri) throws IOException {
```

Comment:
Isn't it possible to make less API calls to get the API?
Calling `exists` or calling `getFileAttributes` and catching the NoSuchKeyException will do the same but with 1 API call less.

Reply:
s3Client.headObject does not throws NoSuchKeyException but a weird AwsException instead (at least with localstack...). So I could not save the call to exists...